src/courtroom.cpp

```diff
@@ -1573,45 +1573,22 @@ void Courtroom::play_sfx()
 
 void Courtroom::set_text_color()
 {
+  QMap<Color, QStringList> all_colors = ao_app->get_chatmessage_colors();
+  Color color = (Color)m_chatmessage[CMTextColor].toInt();
```

Comment:
Don't do C-style casting unless you know what you're doing. In this case, it's also unnecessary.

Reply:
To explain why, C-style casting basically means that you're asking the compiler to convert to whatever you're asking to cast it to. Absolutely anything undefined that could lead to crash, in other words, use other kinds of castings.